x/concentrated-liquidity/internal/math/tick.go

```diff
@@ -50,14 +50,18 @@ func TickToSqrtPrice(tickIndex, exponentAtPriceOne sdk.Int) (price sdk.Dec, err
 	geometricExponentIncrementDistanceInTicks := sdkNineDec.Mul(powTen(exponentAtPriceOne.Neg()))
 
 	// Check that the tick index is between min and max value for the given exponentAtPriceOne
-	minTick, maxTick := GetMinAndMaxTicksFromExponentAtPriceOne(exponentAtPriceOne)
+	minTick, maxTick := getMinAndMaxTicksFromExponentAtPriceOne(exponentAtPriceOne)
 	if tickIndex.LT(sdk.NewInt(minTick)) {
 		return sdk.Dec{}, types.TickIndexMinimumError{MinTick: minTick}
 	}
 	if tickIndex.GT(sdk.NewInt(maxTick)) {
 		return sdk.Dec{}, types.TickIndexMaximumError{MaxTick: maxTick}
 	}
 
+	if tickIndex.Equal(sdk.NewInt(minTick)) {
+		return sdk.ZeroDec(), nil
+	}
+
```

Comment:
We also need to add check for max tick here if thats the case(i suppose)

Reply:
One of the current open question:

>* Do we agree with the logic of replacing the `minTick` with a price of 0 instead of `.000000000001`?
    * This seems like an easy "yes", but I think replacing the maxTick's price from the current max spot price of `100000000000000000000000000000000000000` to whatever the max BigDec is seems more questionable.